### DISCUSSION TITLE
Is there a "warm up" time?

### DISCUSSION THREAD

**Discussion by ydennisy** [supabase/supabase#4495] [General]
Hello!

I am using Prisma to connect to Supabase.

Currently my db is small and all the queries are very simple, what I am noticing is a big difference in the first call VS the next call.

Sorry about the rough anecdotal number but the first call could be 500ms and subsequent calls are 30ms.

This brings me to ask if there is any sort of warm up, caching behaviour implemented on your side? Or anything else which may cause such a difference.

Best,
Dennis

**Comment by burmecia** [comment]
Hi Dennis,

The database is running in a 'always live' mode instance, so I don't think there are any 'warm up' time for it. As regarding the 1st call latency, I think it might due to the DNS resolution. Supabase has a unique DNS entry for each new database, so when you connect it for the first time, it might take some time for the DNS address resolution but will be much faster once it is completed and cached locally.
